apps/settings/elements/bluetooth.html

```diff
@@ -51,7 +51,7 @@ <h2 data-l10n-id="bluetooth-devices-in-area">Devices found</h2>
 
     <form role="dialog" data-type="confirm" id="update-device-name" hidden>
       <section id="device-name-update">
-        <h1 data-l10n-id="change-device-name" id="change-device-name-info">Change My Device</h1>
+        <h1 data-l10n-id="rename-my-device" id="change-device-name-info">Rename My Device</h1>
```

Comment:
This change was done as per David comments, previously we did the same as you mentioned. Please see the David's Comments below: "...es/device_type/phone/settings.device.en-US.properties @@ -2,7 +2,7 @@
 # (phone, tablet, etc.). Try to be consistent across files for each type of
 # device, but don't use a simple copy and paste: string ID is the same, but
 # the device type is different in each file.
-change-device-name = Change phone name
+change-device-name = Rename My Device
@davidflanaganOwner
davidflanagan added a note 7 days ago
A general rule for localization files like this is that you are not allowed to change existing strings. If the text of the string has to change, you must create a new string id. So in this case you could use change-device-name-2 for the new string. Then, of course, any html or JS that uses 'change-device-name' would have to change to 'change-device-name-2'.

If you change the text without changing the id, localizers may not know that a new translation is needed."